<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sudoku_GUIClass</class>
 <widget class="QMainWindow" name="Sudoku_GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sudoku_GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNewGame">
    <property name="title">
     <string>NewGame</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSave">
    <property name="title">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuLoad">
    <property name="title">
     <string>Load</string>
    </property>
   </widget>
   <addaction name="menuNewGame"/>
   <addaction name="menuSave"/>
   <addaction name="menuLoad"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Sudoku_GUI.qrc"/>
 </resources>
 <connections/>
</ui>
